rebasehelper/output_tools/html_output_tool.py

```diff
@@ -0,0 +1,369 @@
+                    with tag('div', klass='well'):
+
+                        for l in f.readlines():
+                            doc.asis('%s<br>' % l)
+
+                os.unlink('error.log')
+
+            except:
+                # No particular log should be displayed
+                pass
+
+        with tag('p'):
+            doc.asis('Please see the whole <a href="./dir_struct.html">Rebase Helper Results directory</a>\
+            for manual file browsing.<br>')
+
+    @classmethod
+    def print_logs(cls):
+        """Wrapper method for printing all logs content to HTML"""
+        if 'builds' not in results_store.get_all():
+            return
+
+        logs = results_store.get_all()['builds']
+
+        for version in ['old', 'new']:
+            if version in logs:
+                cls.page_setup()
+                cls.print_logs_section(logs[version], version)
+                filename = os.path.join(cls.html_path, version+'_logs.html')
+
+                with open(filename, 'w') as f:
+                    f.write(doc.getvalue())
+
+
+    @classmethod
+    def print_logs_section(cls, logs, ver):
+        """
+        Print either all new or all old logs
+
+        :param logs: dictionary containing dictionary of logs
+        :param ver: old/new version string
+        """
+        logger.debug("Creating %s_logs.html" % ver)
+        with tag('hr'):
+            pass
+        with tag('h3'):
+            text('%s version logs' % ver)
+
+        for log in logs['logs']:
+            with tag('div', klass='well'):
+                with tag('h4'):
+                    text(log)
+                cls.write_file_content(log)
+
+
+    @classmethod
+    def print_checkers(cls):
+        """Print checkers content to HTML"""
+        if not 'checkers' in results_store.get_all():
+            return
+        logger.debug("Creating checkers.html")
+
+        cls.page_setup()
+        with tag('h3'):
+            text('Checkers output')
+
+        for checker, checker_dict in six.iteritems(results_store.get_all()['checkers']):
+            with tag('div', klass='well'):
+                with tag('h4'):
+                    text(checker)
+                try:
+                    if checker_dict is not None:
+                        for path in checker_dict:
+                            cls.write_file_content(path)
+                except:
+                    text('%s checker output does not exist' % checker)
+
+
+        with open(os.path.join(cls.html_path, 'checkers.html'), 'w') as f:
+            f.write(doc.getvalue())
+
+    @classmethod
+    def write_file_content(cls, filename):
+        """
+        Print content of a file to HTML
+
+        :param filename: name of a file that will be printed to HTML
+        """
+        logger.debug("Writing content of %s" % filename)
+        try:
+            with open(filename, 'r') as f:
+                for line in f.readlines():
+                    doc.asis(line + '<br>')
+        except:
```

Comment:
You should always catch specific exceptions, here it would be probably `IOError`.